### PR TITLE
fix: fix E2E CI failure

### PR DESCRIPTION
**Pull request by ace-n**

### CODE REVIEW

**Comment by ace-n** [comment]
**Notes:**
1) The `package.json` change is what broke CI; I had to update `npm` and refresh the lockfile.
2) No `EMBLEM_API_URL` was specified for the Cloud Build trigger; I specified one manually to get the build passing.

@rogerthatdev PTAL at the Terraform implementation - thanks!

File: terraform/modules/ops/variables.tf [hunk 1]
```diff
@@ -32,3 +32,8 @@ variable "setup_cd_system" {
   default     = false
   description = "Create deployment triggers. Enable only if Cloud Build has been granted GitHub access."
 }
+
+variable "e2e_api_url" {
```
Comment (rogerthatdev) [inline terraform/modules/ops/variables.tf:36]: Adding a variable to the ops module may cause setup.sh to fail since we don't provide a value for it.

**Review by rogerthatdev — CHANGES_REQUESTED** [review]
For this change to play nice with current setup, it'll need to provide a value for that variable via `setup.sh`

Also, we could consider creating a `emblem-testing` module in `terraform/modules/` so we can keep it separated from the other ops resources and develop it more independently

File: terraform/modules/ops/variables.tf (hunk 1 above)
Comment (ace-n) [reply terraform/modules/ops/variables.tf:36]: Noted - that was intentional, but I made it optional for now based on our internal conversation.

**Comment by ace-n** [comment]
@rogerthatdev I added a few commits as notes for myself; no obligation to review them. 🙂

**Comment by rogerthatdev** [comment]
Closing in favor of PR #719 and #726

**Pull request closed without merging**